clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -23,6 +25,9 @@ import java.time.format.DateTimeFormatter
 import org.json4s.native.JsonMethods._
 import org.json4s._
 import org.json4s.JsonDSL._
+import org.apache.commons.lang3.StringUtils
```

Comment:
Why do we need this?  Scala provides its own StringUtils, _and_ makes it implicit on Strings:
```scala
val s = "the quick brown fox"
// val s: String = the quick brown fox

s startsWith("the qu")
// val res0: Boolean = true
```
(You don't need a dot to apply a method to an object in Scala, so the second line is either more readable or a bit scary...)

Reply:
Removed